ts/features/whatsnew/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { useSelector } from "react-redux";
```

Comment:
Maybe we could consider to split all this file code into multiple subfolders under the `whatsnew` folder? For example:
- `selector`
- `version`
We could leave `types` here if you like. What do you think?